sda-commons-server-spring-data-mongo/src/test/java/org/sdase/commons/server/spring/data/mongo/SpringDataMongoBundleSSLConfigurationIT.java

```diff
@@ -44,7 +45,7 @@ class SpringDataMongoBundleSSLConfigurationIT {
           null,
           randomPorts(),
           config("springDataMongo.connectionString", mongo::getConnectionString),
-          config("springDataMongo.useSsl", "false"),
+          config("springDataMongo.useSsl", "true"),
```

Comment:
This value does not match the use case, does it? The app extension's name is `DW_WITHOUT_SSL` but here you set the value to `true`.

I would prefer to not set this deprecated property anymore at all. If the connection string is configured it should override all other settings IMHO.

Reply:
> If the connection string is configured it should override all other settings IMHO.

That's why I changed the test. But I'm not sure if this assumption is correct in all cases as I don't know if the mongo client will upgrade to SSL automatically, if it is not set in the connection string, but the server only offers SSL.